packages/block-library/src/block/editor.scss

```diff
@@ -1,35 +0,0 @@
-.edit-post-visual-editor .block-library-block__reusable-block-container {
```

Comment:
These styles target reusable blocks in non-iframe mode. In my testing reusable blocks still behave and look properly without these (just like they do in iframe mode)